sdk/monitor/Azure.Monitor.Query/CHANGELOG.md

```diff
@@ -7,8 +7,11 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Enable national cloud support for US Gov and China clouds
```

Comment:
You are releasing a beta before GA and then a beta after with everything? Any reason why you are not doing a GA directly and then only a beta after with everything?

Reply:
We are just doing a minor version bump here so this package would release as 1.3.0. GA will come later (hopefully in March). There is no beta in the middle.

Reply:
You will need to update the changelog and csproj to reflect the GA version.

Reply:
This will not ship right now - this will ship with the MetricsBatch GA in March.